honeycombio/resource_trigger_test.go

```diff
@@ -79,6 +81,20 @@ func testAccCheckTriggerAttributes(t *honeycombio.Trigger) resource.TestCheckFun
 			return fmt.Errorf("bad frequency: %d", t.Frequency)
 		}
 
+		if t.AlertType != "on_change" {
+			return fmt.Errorf("bad AlertType: %s | should be on_change", t.AlertType)
+		}
+
+		return nil
+	}
+}
+
+func testAccCheckTriggerAlertType(t *honeycombio.Trigger) resource.TestCheckFunc {
```

Comment:
By using the ValidateFunc above this won't be needed

Reply:
awesome, will remove testAccCheckTriggerAlertType then